notebook/static/notebook/js/outputarea.js

```diff
@@ -682,36 +682,43 @@ define([
     OutputArea.prototype.append_mime_type = function (json, element, handle_inserted) {
         for (var i=0; i < OutputArea.display_order.length; i++) {
             var type = OutputArea.display_order[i];
-            var append = OutputArea.append_map[type];
```

Comment:
Can the try/catch be a little narrower? Where does this typically raise, in the `toinsert = append.apply`?

Reply:
Yes, it's `append.apply` that would throw if https://github.com/jupyter-widgets/ipywidgets/pull/3280 gets merged, I will make it narrower. Thanks for the review!